pkg/actions/karpenter/tasks.go

```diff
@@ -1,11 +1,12 @@
 package karpenter
 
 import (
+	"context"
 	"fmt"
 	"sort"
 
+	cfntypes "github.com/aws/aws-sdk-go-v2/service/cloudformation/types"
```

Comment:
ah you see, like here. :D

Reply:
I had to here! Multiple types package. if we can avoid an alias its easier, also i'm too comitted to not aliasing now :joy: